src/api/browse.ts

```diff
@@ -1,5 +1,28 @@
 import { searchResponseModel, datasetEmbeddedModel, datasetSummaryModel } from "../models/dataset";
 import { facetFilterModel } from "../models/facets";
+import authService from "../components/auth/authService";
```

Comment:
I think we can move the authService either to api or introduce a folder called service and move it there. In general, the components folder only contains the UI elements in the hierarchy we see on the UI.

Reply:
Yes, the auth service is not an actual react component. Will refactor this.